src/Fixer/DeprecatedFixerInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer;
+
+/**
+ * @author Kuba Werłos <werlos@gmail.com>
```

Comment:
@SpacePossum @juliendufresne 
we don't want ppl to implement this interface, for that it shall be internal.
or, we want to allow ppl the programatically check if given concrete fixer follows this interface?

Reply:
I would keep it public, not `@internal`. This can be useful for people that use this project as a lib to deprecate their own fixers.

Reply:
I agree with @julienfalque here, although I've no strong opinion on it, but it could be nice for 3rd party fixer providers to deprecate their fixers as well using the mechanic.

Reply:
thanks for opinion :+1: let it stay public